src/components/POS/Classic/ItemsTable.vue

```diff
@@ -29,7 +29,10 @@
     </div>
   </Row>
 
-  <div class="overflow-y-auto" style="height: 80vh">
+  <div
+    class="overflow-y-auto custom-scroll custom-scroll-thumb2"
+    style="height: calc(70vh)"
```

Comment:
using `calc` is unnecessary here unless you're performing a calculation like `calc(2em * 5);`